### ISSUE TITLE
[Anon] Bug report: Grey Knights - Codex (2014).catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#1417]
**File:** Grey Knights - Codex (2014).catz

**BattleScribe version:** 1.15.07

**Platform:** Android

**Dropbox:** Yes

**Description:** Hi, according to forge world (I sent them an email) psybolt, psyflame and psychic pilot options are still used for the Land Raider Vortimer. Same goes for all FW Grey Knight Models.

**Comment by Itsacon** [comment]
Psyflame is still included, (although the rules should be listed in the datafile, I'll see about fixing that)

Psychic pilot should use the rules in the new BRB, as the FW rules assume a different psychic phase.

Psybolt is impossible to use, as IA2:SE refers to Codex:Grey Knights where rules are concerned, and C:GK doesn't contain any reference to Psybolt ammunition anymore. Until they put out a new Errata, Psybolt ammo has no rules, and while technically, you could spend 5 points on it, I don't think it's useful to include it in the datafiles.

**Comment by Itsacon** [comment]
Ok, I've updated and added some of the special rules, but to be frank, the GK datafile needs a lot of work in the FW department. 

Basically, all the psychic pilot stuff is weird. The powers listed in IA2 don't have Warp Charges, so they are unusable. Does this mean the vehicles just get Psychic Pilot, and draw their powers like other psykers? 

Then there's the wargear options without rules like Psybolt ammo, or rules that make no sense like Reinforced Aegis. (And what about the Aegis itself? Do the vehicles use the FW rule, or the more recent codex version?)

Basically, FW really needs to put out an FAQ/Errata for the GK vehicles, and until then, anything we do is blatant guesswork, and likely to conflict with someone's interpretation.
